assets/stylesheets/style.scss

```diff
@@ -1,85 +1,150 @@
 $green: #1FB682;
 
+//GENERAL
+
+html {
+  overflow: auto
+}
+
+body {
+  height: 100%;
+  overflow: hidden;
+  width: 100%;
+}
+
 * {
+	-moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
   box-sizing: border-box;
-  font-family: "Poppins";
   color: #504949;
+  font-family: "Poppins";
   font-weight: 300;
-  -webkit-font-smoothing: antialiased;
-	-moz-osx-font-smoothing: grayscale;
 }
 
-#main {
-  overflow: hidden
-}
-
-
-//TEXT STYLES
-.text.-caps {
-  text-transform: uppercase;
+h1, h2, h3, h4 {
+  margin-top: 0;
 }
 
-.centered {
+.heading-container {
   text-align: center;
 }
 
-
-.text.-bold {
-  font-weight: 600;
-}
-
-.text.-wide-spacing {
-  letter-spacing: .23em;
+.sr-only {
+  clip: rect(1px 1px 1px 1px); /* IE 6/7 */
+  clip: rect(1px, 1px, 1px, 1px);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
+
+.conversation__icon,
+.conversation__background,
+.flo-diagram,
+.meet-flo__img,
+.footer__bg,
+.footer__img,
+.phone,
+.pills {
+  background-repeat: no-repeat;
+  background-size: contain;
+  position: absolute
 }
 
-.text.-serif {
-  font-family: 'Merriweather';
-  font-weight: 300;
-}
 
-.text.-x-small {
-  font-size: 14px;
-  line-height: 1.733333333;
-}
+//TEXT STYLES
 
-.text.-small {
+//just experimenting with @extend
+%text-small {
   font-size: 18px;
   line-height: 1.777777778;
 }
 
-.text.-medium {
-  font-size: 20px;
-  line-height: 1.428571429;
+%text-caps {
+  text-transform: uppercase;
 }
 
-.text.-large {
-  font-size: 28px;
-  line-height: 1.428571429;
+%text-bold {
+  font-weight: 600;
 }
 
-.text.-color-dark {
-  color:#0e5254;
+%text-color-accent {
+  color: #BB7140;
 }
 
-.text.-color-lightblue {
-  color: #1394AA;
-  margin-bottom: 30px
+.text--subheading {
+  @extend %text-small;
+  @extend %text-caps;
+  @extend %text-bold;
+  @extend %text-color-accent;
 }
 
-.text.-color-white {
-  color:white;
-}
+.text {
+  &.-caps {
+    @extend %text-caps;
+  }
 
-.text.-color-accent {
-  color: #BB7140;
+  &.-centered {
+    text-align: center;
+  }
+
+  &.-bold {
+    @extend %text-bold
+  }
+
+  &.-wide-spacing {
+    letter-spacing: .23em;
+  }
+
+  &.-serif {
+    font-family: 'Merriweather';
+    font-weight: 300;
+  }
+
+  &.-x-small {
+    font-size: 14px;
+    line-height: 1.733333333;
+  }
+
+  &.-small {
+    @extend %text-small;
+  }
+
+  &.-medium {
+    font-size: 20px;
+    line-height: 1.428571429;
+  }
+
+  &.-large {
+    font-size: 28px;
+    line-height: 1.428571429;
+  }
+
+  &.-color-dark {
+    color:#0e5254;
+  }
+
+  &.-color-lightblue {
+    color: #1394AA;
+    margin-bottom: 30px
+  }
+
+  &.-color-white {
+    color:white;
+  }
+
+  &.-color-accent {
+    @extend %text-color-accent;
+  }
 }
 
-cite a{
+cite a {
   color: #EC4444;
+  font-size: 13px;
+  font-style: normal;
   text-decoration: underline;
```

Comment:
I think generally we want to keep underlines on **all** `a` tags and only remove the underline. I am assuming you are adding it back in here because it's globally taken out (but I could be wrong). Just something to note re: a11y.